test/test_importing.py

```diff
@@ -40,3 +40,11 @@ def test_importing_attributes(self):
         assert mod.__spec__.loader is mod.__loader__
         assert mod.__spec__.name == 'mytest'
         assert mod.__file__
+
+        if hpy_abi == 'cpython':
+            from distutils.sysconfig import get_config_var
+            ext = get_config_var('EXT_SUFFIX')
+        else:
+            ext ='.hpy.so'
+
```

Comment:
this is probably wrong on windows (and maybe mac), but I think it's fine not to care until we actually start to run tests on those platforms.

Reply:
I'm currently developing on a mac,  it's fine:
```
In [3]: sysconfig.get_config_var("EXT_SUFFIX")
Out[3]: '.cpython-39-darwin.so'
```

 On windows it should probably be ".pyd", but it also needs to be fixed in https://github.com/hpyproject/hpy/blob/master/hpy/devel/__init__.py#L248